woocommerce/woocommerce
# Bump WCA JS packages version numbers in package.json and changelogs

### All Submissions:


### Changes proposed in this Pull Request:

- Updated WCA npm packages publishing commands in WCA's package.json
- Bumped WCA JS packages version in package.json and changelogs to prepare for publishing

### How to test the changes in this Pull Request:

1. Run `pnpm run publish-packages:check --filter @woocommerce/admin-library` in repo root
Expect success and to see output:

```
{
        "publishedPackages": [
                {
                        "name": "@woocommerce/csv-export",
                        "version": "1.5.0"
                },
                {
                        "name": "@woocommerce/date",
                        "version": "4.0.1"
                },
                {
                        "name": "@woocommerce/dependency-extraction-webpack-plugin",
                        "version": "2.0.0"
                },
                {
                        "name": "@woocommerce/eslint-plugin",
                        "version": "2.0.0"
                },
                {
                        "name": "@woocommerce/explat",
                        "version": "2.1.0"
                },
                {
                        "name": "@woocommerce/navigation",
                        "version": "7.0.1"
                },
                {
                        "name": "@woocommerce/number",
                        "version": "2.2.1"
                },
                {
                        "name": "@woocommerce/tracks",
                        "version": "1.1.1"
                },
                {
                        "name": "@woocommerce/admin-e2e-tests",
                        "version": "1.0.0"
                },
                {
                        "name": "@woocommerce/currency",
                        "version": "4.0.1"
                },
                {
                        "name": "@woocommerce/data",
                        "version": "3.1.0"
                },
                {
                        "name": "@woocommerce/components",
                        "version": "10.0.0"
                },
                {
                        "name": "@woocommerce/experimental",
                        "version": "3.0.1"
                },
                {
                        "name": "@woocommerce/customer-effort-score",
                        "version": "2.0.1"
                },
                {
                        "name": "@woocommerce/onboarding",
                        "version": "3.0.1"
                }
        ]
}
```

### FOR PR REVIEWER ONLY:
